src/pages/AddListingPage/components/BasicDetailsStep.tsx

```
@@ -154,23 +157,23 @@ export const BasicDetailsStep: React.FC<BasicDetailsStepProps> = ({
             <div className="flex flex-col items-center justify-center text-center leading-[0]">
-              <p className="font-['Poppins'] text-sm leading-normal text-[#ef4444]">
+              <Text className="font-['Poppins'] text-sm leading-normal text-[#ef4444]">
                 *
-              </p>
+              </Text>
             </div>
           </div>
           <div
             className="flex shrink-0 cursor-pointer items-center justify-center gap-2"
             onClick={onTipsClick}
           >
```

Biome (2.3.13)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)